src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
Consider using a consistent margin direction for your elements. You have `margin-right` here, but the checklist recommends adding margins on only one axis (top or bottom for vertical rhythm) to avoid potential margin collapse.